src/main/scala/beam/utils/map/NewYorkRouteDebugging.scala

```diff
@@ -0,0 +1,309 @@
+package beam.utils.map
```

Comment:
Should these App level classes be stored in a separate location so they are easier to discover?

Reply:
I am not sure, if it should, since Art is not with us anymore and I am not the author of these files. Should we leave it like this or move them somewhere? If we should, please list the files that need to be moved and also propose the new location. @JustinPihony

Reply:
I created https://github.com/LBNL-UCB-STI/beam/issues/3445